pyproject.toml

```
@@ -0,0 +1,48 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+
+# Testing tools configuration
+[tool.coverage.run]
+branch = true
+
+[tool.coverage.report]
+show_missing = true
+
+[tool.pytest.ini_options]
+minversion = "6.0"
+log_cli_level = "INFO"
+
```

I think this solves a "in the future, this is going to break" warning in CI.  But I don't recall the details and might be wrong about it

```suggestion
log_cli_level = "INFO"
asyncio_mode = "auto"

```